cmd/main.go

```diff
@@ -34,8 +34,10 @@ func main() {
 	var err error
 	var config *pkg.Configuration
 
+	//TODO When Vault Integration is added the check: if env == "" should be removed! It is temporary added for sake of Snap builds.
+	//The default behaviour in case secretStore is not present is to read the credentials from Vault
 	switch env := os.Getenv(pkg.SecretStore); {
```

Comment:
you can do this a bit cleaner with `os.LookupEnv` like:

```go
env, exists := os.LookupEnv(pk.SecretStore)
switch {
    !exists || env == "false":
        // load insecure config
}
```

Reply:
Thank you